css/style.css

```diff
@@ -0,0 +1,30 @@
+nav {
```

Comment:
@dbklein6 LOOOOOVE the fact that you're using 2-space indentation. To each is own, but this is my personal preference :)